web-marketplace/src/components/molecules/CreditsAmount/CurrencyInput.tsx

```diff
@@ -77,15 +77,17 @@ export const CurrencyInput = ({
   return (
     <div className="grow sm:flex-1 w-full sm:w-auto relative">
       {paymentOption === PAYMENT_OPTIONS.CARD && (
-        <span className="absolute top-[18px] left-10 z-50">$</span>
+        <span className="text-xs sm:text-sm absolute top-[13px] left-[9px] sm:top-[18px] sm:left-10 z-50">
+          $
+        </span>
       )}
       <TextField
         {...register(CURRENCY_AMOUNT)}
         onChange={handleOnChange}
         type="number"
         className={`border border-solid border-grey-300 focus-within:border-grey-500 focus-within:border-2 ${
           paymentOption === PAYMENT_OPTIONS.CARD ? 'pl-5' : ''
-        } w-full sm:w-auto flex justify-start relative sm:h-60 rounded-sm items-center`}
+        } w-full sm:w-auto flex justify-start relative xs:h-60 rounded-sm items-center`}
```

Comment:
we don't have `xs` breakpoint so `xs:h-60` doesn't do anything

Reply:
That was a mistake, thanks for catching it. I’ll revert it back.